lmfdb/genus2_curves/main.py

```diff
@@ -290,6 +289,45 @@ def genus2_lookup_equation(f):
             return r['label']
     return None
 
+
+def geom_inv_to_G2(inv):
+    def igusa_clebsch_to_G2(Ilist):
+        # first to Igusa
+        I2, I4, I6, I10 = Ilist
+        J2 = I2/8
+        J4 = (4*J2**2 - I4)/96
+        J6 = (8*J2**3 - 160*J2*J4 - I6)/576
+        J8 = (J2*J6 - J4*2)/4 # we won't use this one at all
```

Comment:
Doesn't match the knowl, should be J4**2 right? Did you test this code against the db, it should be possible to run through a bunch of test cases there?

Reply:
Oh lol this is the unused one, so you probably did test it :D